src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -233,7 +233,7 @@ describe('workspace service', () => {
         })
         .expect(200);
       // Global and Management workspace will be created by default after workspace list API called.
```

Comment:
Could you please remove the `Management` in the comment?

Reply:
done